CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,2 @@
+- Upgrade from node-uuid ~1.4.1 to uuid ~3.3.2
```

Comment:
This was only an example to show style. In fact, it is not correct this time as uuid didn't change in pacakges.json in this PR. Sorry for not being clearer...

I'll try to provide a real example for a dependency and another for dev dependency:

```
- Upgrade async dependency from 2.6.1 to 2.6.2
- Upgrade mocha dev dependency from 5.2.0 to 6.1.4
```

```
```

All changes in packages.json dependencies and dev dependencies should be included this way in CHANGES_NEXT_RELEASE.

Reply:
Fixed 50476d3